scripts/revive.toml

```diff
@@ -0,0 +1,27 @@
+# Enabled rules
+[rule.blank-imports]
+[rule.context-as-argument]
+[rule.context-keys-type]
+[rule.dot-imports]
+[rule.error-return]
+[rule.error-strings]
+[rule.error-naming]
+[rule.exported]
+[rule.increment-decrement]
+[rule.var-naming]
+[rule.var-declaration]
+[rule.package-comments]
+[rule.range]
+[rule.receiver-naming]
+[rule.time-naming]
+[rule.unexported-return]
+[rule.indent-error-flow]
+[rule.errorf]
+[rule.superfluous-else]
+[rule.unused-parameter]
+[rule.unreachable-code]
+[rule.redefines-builtin-id]
+
+# Disabled rules
+[rule.empty-block]
+    Disabled = true
```

Comment:
Since we have enough effective usages of empty-block, imo we should not bother about it at all. 

```suggestion
```

Reply:
Instead can we add explicit disable for the fixes that are in flight. That way we can rebase this change, and remove that check as we fix them. Seems cleaner to me. That way we can avoid having to see all the linter error annotations on PRs

wdyt?

Reply:
I like the idea. i modified to disable the rules that are yet to merged. Let me know if that looks good